src/processing/dom/index.js

```diff
@@ -297,47 +320,16 @@ export default class DomProcessor {
         this.adapter.removeAttr(el, 'integrity');
     }
 
-    _processJsAttr (el, attr, jsProtocol) {
-        const storedUrlAttr = this.getStoredAttrName(attr);
-        const processed     = this.adapter.hasAttr(el, storedUrlAttr);
-        const attrValue     = this.adapter.getAttr(el, processed ? storedUrlAttr : attr);
-
-        const code    = jsProtocol ? attrValue.replace(JAVASCRIPT_PROTOCOL_REG_EX, '') : attrValue;
-        const matches = code.match(HTML_STRING_REG_EX);
+    _processJsAttr (el, attr, isJsProtocol, isEventAttr) {
```

Comment:
change the method signature as `_processJsAttr ({ el, attr, isJsProtocol, isEventAttr })`